### DISCUSSION TITLE
Automatically Replace templates in the active file when a template is inserted using "Template: Insert template"

### DISCUSSION THREAD

**Discussion by violet-fish** [SilentVoid13/Templater#1240] [Help] [answered]
I would like to insert a template with some templater code to automatically move it to the correct directory. 

I created a large number of new files that don't exist and would like to format them all with a template and move them to the correct directory (as part of the template). I have a template that inserts all the raw text I would like and moves the file, but when I use "Template: Insert template", it inserts the template without running the templater code to move the file to the correct directory.

### Current workflow
1. Create file by clicking on the link to an uncreated note
2. Insert template (Template: Insert template)
3. Replace templates in active file

### Desired workflow
1. Click uncreated note name
2. Insert template and replace templates in file

Alternatively:
1. Create new note from template linked to uncreated note name

I found #906 asking a similar problem, but I have already created the note when the template is inserted. I have 50 notes I will need this template run on, and would like to reduce the amount of manual intervention required

**Comment by Zachatoo** [comment, marked as answer]
Sounds like you're running the core Templates command to insert a template instead of the Templater command. The Templater command is `Templater: Open Insert Template Modal`.

If you aren't using the core Templates plugin, I recommend disabling it, as it's easy to confuse them.

**Reply by violet-fish** [reply]
Thanks, I didn't expect inserting and running a template to be named that

**Comment by Cryptodarktrader** [comment]
If what you want is for your template to have a code that moves the note to the correct folder, I use this:



```
<%*
//🔀 Move the note to the corresponding folder.
await tp.file.move(`/20 - 🍜 Recipes/${Title}`);
%>
```
You just have to leave the "/" and put the address after that.
